pkg/apis/serving/v1alpha1/revision_conversion.go

```diff
@@ -46,11 +46,12 @@ func (source *RevisionTemplateSpec) ConvertUp(ctx context.Context, sink *v1beta1
 
 // ConvertUp helps implement apis.Convertible
 func (source *RevisionSpec) ConvertUp(ctx context.Context, sink *v1beta1.RevisionSpec) error {
-	sink.ContainerConcurrency = v1beta1.RevisionContainerConcurrencyType(
-		source.ContainerConcurrency)
 	if source.TimeoutSeconds != nil {
 		sink.TimeoutSeconds = ptr.Int64(*source.TimeoutSeconds)
 	}
+	if source.ContainerConcurrency != nil {
+		sink.ContainerConcurrency = ptr.Int64(*source.ContainerConcurrency)
```

Comment:
`GetContainerConcurrency()`?

Reply:
I explicitly commented on having it this way around (same as TimeoutSeconds). Do we really want to put inferred values in when doing a conversion?

Reply:
I missed the nil check, seems fine.